documentation/src/main/asciidoc/topics/proc_configuring_sql_stores_query.adoc

```diff
@@ -0,0 +1,52 @@
+[id='configuring-sql-cache-stores-query_{context}']
+= Configuring SQL query stores
+
+Add SQL query cache stores so {brandname} can preload your caches from database tables.
+
+.Prerequisites
+
+* For remote caches, copy any database drivers to the `server/lib` directory in your {brandname} Server installation.
+
+.Procedure
+
+. Create a SQL cache store configuration in one of the following ways:
++
+* Declaratively, add the `persistence` element or field then add `query-jdbc-store` with the following schema namespace:
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+xmlns="urn:infinispan:config:store:sql:{schemaversion}"
+----
++
+* Programmatically, add the following methods to your `ConfigurationBuilder`:
++
+[source,xml,options="nowrap",subs=attributes+]
+----
+persistence().addStore(??)
```

Comment:
@wburns Are there dependencies you need for the SQL store? I couldn't find them for 13. How do you add a SQL store like this?